node/libs/roles/src/attester/keys/batch_public_key.rs

```diff
@@ -0,0 +1,37 @@
+use std::fmt;
+use zksync_consensus_crypto::{bn254, ByteFmt, Text, TextFmt};
+
+use crate::validator::PublicKey;
+
+/// A public key for a validator used in L1 batch signing.
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct BatchPublicKey(pub(crate) bn254::PublicKey);
+
+impl ByteFmt for BatchPublicKey {
```

Comment:
please remove the "Batch" prefix from PublicKey, SecretKey, Signature. The fact that attester key can be used to sign L1Batches is the property of the attester message.

Reply:
+1